src/Controls/src/Core/View.cs

```diff
@@ -193,43 +189,5 @@ void ValidateGesture(IGestureRecognizer gesture)
 			if (gesture is PinchGestureRecognizer && _gestureRecognizers.GetGesturesFor<PinchGestureRecognizer>().Count() > 1)
 				throw new InvalidOperationException($"Only one {nameof(PinchGestureRecognizer)} per view is allowed");
 		}
-
-		#region IView
-
-		protected PropertyMapper propertyMapper;
```

Comment:
Moved these 2 regions into the `View.Impl.cs` file as that is where this should be.